.gitignore

```diff
@@ -1,3 +1,8 @@
+# Rendered templates
+deploy/025_sourcecode.yaml
```

Comment:
I'm seeing manifests/ as the stdd name for this folder in other operators/components

Maybe it would make sense to rename from deploy.

Reply:
Recapping here for the benefit of folks not in the Slack group:

We're leaving `deploy` for the time being until the landscape around us matures a little more. This might end up changing to `manifests` or something else entirely.

Reply:
lgtm, makes sense